README.md

```diff
@@ -19,217 +19,172 @@
+runtimes or any other high-level user-space concepts. In order to relate this data to these high-level
+concepts and make the eBPF data immediately more understandable, Inspektor Gadget automatically
+uses kernel primitives such as mount namespaces, pids, etc. to infer which high-level
+concepts they relate to; Kubernetes pods, container names, DNS names, etc. The process of augmenting
+the eBPF data with these high-level concepts is called *enrichment*.
+
+Enrichment flows the other way, too. Inspektor Gadget enables users to do high-performance
+in-kernel filtering by only referencing high-level concepts such as Kubernetes pods, container
+names, etc.; automatically translating these to the corresponding low-level kernel resources.
+
+See the [enrichment documentation](docs/enrichment)
+
+### What is an operator
+
+In Inspektor Gadget, an operator is any part of the framework where an action is taken. Some operators are under-the-hood (i.e. fetching and loading Gadgets) while others are user-exposed (enrichment, filtering, export, etc.) and can be reordered and overridden.
 
-Inspektor Gadget is deployed to each node as a privileged DaemonSet.
-It uses in-kernel eBPF helper programs to monitor events mainly related to
-syscalls from userspace programs in a pod. The eBPF programs are run by
-the kernel and gather the log data. Inspektor Gadget's userspace
-utilities fetch the log data from ring buffers and display it. What eBPF
-programs are and how Inspektor Gadget uses them is briefly explained in
-the [architecture](docs/core-concepts/architecture.md) document.
+See the [operator documentation](docs/operators.md) for more information.
+
+### Further learning
+
+Use the [project documentation](/docs/_index.md) to learn more about:
+
+* [Overview](/docs/_index.md)
+* [Concepts](/docs/core_concepts/_index.md)
+* [Components](https://inspektor-gadget.io/docs/components)
+* [Gadgets](https://inspektor-gadget.io/docs/gadgets)
+* [Reference](https://inspektor-gadget.io/docs/reference)
+* [Contributing]()
+
+## Installation
+
+Inspektor Gadget can be installed in several different ways depending on your environment and needs.
+
+### On Linux
+
+This installs the `ig` and `gadgetctl` tools
+
+TODO: sudo dnf install `inspektor-gadget`
+
+### For Kubernetes
+
+Use the [krew](https://sigs.k8s.io/krew) plugin manager to install the gadget `kubectl` plugin and
+deploy Inspector Gadget into a Kubernetes cluster.
+
+```bash
+kubectl krew install gadget
+kubectl gadget deploy
+```
+
+### On MacOS
+
+This installs the `gadgetctl` client tool which enables communicating with `ig` running in daemon
+mode on a Linux host.
+
+```bash
+brew install ig
+```
+
+### On Windows
+
+This installs the `gadgetctl` client tool which enables communicating with `ig` running in daemon
+mode on a Linux host.
+
+```bash
+# if using Chocolatey
+choco install inspektor-gadget
+# or if using WinGet
+winget install inspektor-gadget
```

Comment:
Yep, trying to point out that this is what the experience should be and if any one wants to tackle that it would be welcome. The Headlamp has experience with this.